Source/WebCore/fileapi/Blob.cpp

```diff
@@ -288,19 +289,38 @@ void Blob::text(Ref<DeferredPromise>&& promise)
     });
 }
 
+static ExceptionOr<JSC::ArrayBuffer&> arrayBufferFromBlobLoader(BlobLoader& blobLoader)
+{
+    if (auto optionalErrorCode = blobLoader.errorCode())
+        return Exception { *optionalErrorCode };
+    RefPtr arrayBuffer = blobLoader.arrayBufferResult();
+    if (!arrayBuffer)
+        return Exception { ExceptionCode::InvalidStateError };
+    return *arrayBuffer;
```

Comment:
Return a reference is a bit dangerous.
In theory, it could return a dead reference if blobLoader or child was not keeping a ref to the array buffer.
Let's return ExceptionOr<Ref<JSC::ArrayBuffer>> or ExceptionOr<RefPtr<>> if the binding generator prefers